.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Coverage
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+
+  build-and-test:
+    name: coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: install seirmo
+        run: |
+          python --version
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install .
+          python -m pip install coverage codecov
+      - name: run coverage
+        run: |
+          coverage run run-tests.py --unit
+      - name: codecov
+        if: success()
+        env:
+          CODECOV_TOKEN: ${{ f4a14d78-425e-41b2-b28e-0a6d2fe886d7 }}
```

Comment:
Could you please generate a new codecov token, store it in the github secrets and then refer to the token here, so the token is not publicly visible? Let me know if you need help with that.

Reply:
OK! Thank you. I’ll make those changes and ask you for help if I can’t😁.